plugins/usbdmx/DMXCProjectsNodleU1.cpp

```diff
@@ -54,6 +54,7 @@ static const int CONFIGURATION = 1;
 static const int INTERFACE = 0;
 
 static const unsigned int DATABLOCK_SIZE = 33;
+static const unsigned int DATABLOCK_MAX_SIZE = 64;
```

Comment:
I don't know why the Nodle U1 (and clones) are limited to 32 bytes. USB and HID both allow 64 bytes (for `bulk transfers` of `full speed` devices). Maybe it's a limitation of Nodle's hardware?
Since the RP2040 doesn't have this limitation, we can send max 64 byte at once